src/controller/java/src/chip/devicecontroller/model/NodeState.java

```diff
@@ -63,7 +61,7 @@ private void addAttribute(
     clusterState.getAttributeStates().put(attributeId, attributeStateToAdd);
   }
 
-  private void addEvent(int endpointId, long clusterId, long eventId, EventState eventStateToAdd) {
+  public void addEvent(int endpointId, long clusterId, long eventId, EventState eventStateToAdd) {
```

Comment:
please set this as protected, so that ReportCallbackJni.Java can still can it from same package, and other package cannot see this.

Reply:
To solve this problem, we modified the NodeState method to be called directly from jni.